doc/templates/Makefile

```diff
@@ -98,7 +98,7 @@ TABLES = Exif \
          OlympusFi \
          OlympusFe1 \
          OlympusRi \
-	 Panasonic \
+	     Panasonic \
```

Comment:
Ups, this line is not aligned properly. It seems the issue is a tab character at the beginning.

Reply:
You're right.  If I was granted one wish, I'd want the tab character to be made illegal in all source code, scripts, makefiles and documentation.  It's a horror.  An invisible character without a definition that can end up on every line of code.